### PR TITLE
[Restructure/plasma] restructure the Radiationfields 

### PR DESCRIPTION
**Pull request by wkerzendorf**
Have to merge #2609

### CODE REVIEW

File: docs/physics/nlte/nlte_rates.ipynb
Comment (andrewfullard) [inline docs/physics/nlte/nlte_rates.ipynb:1]: These notebooks are good but not referenced in any contents file

File: tardis/model/parse_input.py [hunk 1]
```diff
@@ -545,9 +545,7 @@ def parse_radiation_field_state(
 
     assert len(dilution_factor) == geometry.no_of_shells
 
-    return DiluteBlackBodyRadiationFieldState(
-        t_radiative, dilution_factor, geometry
-    )
+    return DilutePlanckianRadiationField(t_radiative, dilution_factor, geometry)
```
Comment (andrewfullard) [inline tardis/model/parse_input.py:548]: This is probably why black is failing (because our Ruff config isn't 80 lines)

File: test.txt
Comment (andrewfullard) [inline test.txt:1]: Delete this

File: astropy_helpers
Comment (andrewfullard) [inline astropy_helpers:1]: Delete this

File: test.txt
Comment (andrewfullard) [inline test.txt:1]: Delete this

**Review by andrewfullard — CHANGES_REQUESTED** [review]

File: tardis/plasma/properties/plasma_input.py [hunk 2]
```diff
@@ -33,8 +35,11 @@ class TRadiative(ArrayInput):
     outputs = ("t_rad",)
     latex_name = (r"T_{\textrm{rad}}",)
 
+    def calculate(self, dilute_planckian_radiation_field):
+        return dilute_planckian_radiation_field.temperature.cgs.value
```
Comment (andrewfullard) [inline tardis/plasma/properties/plasma_input.py:39]: Sure we want a value here and not just the quantity?

**Review by Rodot- — APPROVED** [review]

File: tardis/plasma/properties/plasma_input.py (hunk 2 above)
Comment (wkerzendorf) [reply tardis/plasma/properties/plasma_input.py:39]: currently plasma needs floats.

**Review by Rodot- — APPROVED** [review]
Everything looks good!

**Review by andrewfullard — DISMISSED** [review]

**Comment by andrewfullard** [comment]
Just need to run black please

**Pull request merged**